test/verifiedTests/testCardOpt/testSparseLP.m

```diff
@@ -6,18 +6,13 @@
 % Authors:
 %     - CI integration: Laurent Heirendt March 2017
 %
-% Note:
-%     - The solver libraries must be included separately
-
-% define global paths
-global path_TOMLAB
-global path_GUROBI
 
 % save the current path
 currentDir = pwd;
 
 % initialize the test
-initTest(fileparts(which(mfilename)));
+fileDir = fileparts(which('testSparseLP'));
```

Comment:
wouldnt mfilename instead of the actual fileName make sense here (avoids problems if the file ever gets renamed)